test/test_entrypoint.py

```diff
@@ -46,7 +46,7 @@ def test_trivial_argparse():
 
 
 def test_bad_workspace(tmpdir):
-    args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu', '-d', 'foxy'])
+    args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu', '-d', 'galactic'])
```

Comment:
This doesn't seem necessary to change - the test against foxy is still valid (and will stay valid for a year longer)

Reply:
yeah that's ok. Jut thought we want to keep testing the latest stable.